ivy/functional/frontends/jax/numpy/statistical.py

```diff
@@ -23,6 +23,30 @@ def einsum(
     return ivy.einsum(subscripts, *operands, out=out)
 
 
+@to_ivy_arrays_and_back
+def histogram_bin_edges(a, bins=10, range=None, weights=None):
+    del weights
```

Comment:
May be this could be removed as having this variable doesn't effect the function